modules/brushingandlinking/include/modules/brushingandlinking/brushingandlinkingmanager.h

```diff
@@ -53,13 +53,35 @@ class Deserializer;
 /**
  * Manages brushing and linking events for filtering, selecting, and highlighting. When initialized
  * with a BrushingAndLinking inport, changes are propagated using this port if connected.
+ *
+ * Use setInvalidateOn flags if you only want Processor::process to be called for a subset of
```

Comment:
Any reason for not considering targets? I would get that it is also as common to only care about row or columns for example.

Reply:
Good point. I guess that would be a valid filter and perhaps not as uncommon as I originally thought.